packages/template-retail-react-app/app/hooks/use-search-params.js

```diff
@@ -71,10 +74,13 @@ export const parse = (searchParamsStr, parseRefine = true) => {
     const params = queryString.parse(searchParamsStr, PARSE_OPTIONS)
 
     // Ensure the refinments is an array (make it easier to manipulate).
-    params.refine = Array.isArray(params.refine) ? params.refine : [params.refine]
+    params.refine = Array.isArray(params.refine)
+        ? params.refine
+        : [params.refine].filter((v) => Boolean(v))
```

Comment:
```suggestion
        : [params.refine].filter(Boolean)
```
`Boolean` is already a method that takes a single param. 😉